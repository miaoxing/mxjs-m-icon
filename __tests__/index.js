import Icon from '..';
import {render} from '@testing-library/react';

describe('Icon', () => {
  test('basic', () => {
    const {container} = render(<Icon type="share"/>);
    expect(container).toMatchSnapshot();
  });
});
