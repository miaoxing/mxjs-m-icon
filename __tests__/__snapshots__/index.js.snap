// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon basic 1`] = `
<div>
  <taro-text-core
    class="mxi mxi-share"
  />
</div>
`;
